142. Seth AK, Bayne T. Theories of consciousness. Nat Rev Neurosci (2022) 23
(7):439 –52. doi: 10.1038/s41583-022-00587-4
143. Insel TR, Landis SC, Collins FS. The NIH BRAIN initiative. Science (2013) 340
(6133):687 –8. doi: 10.1126/science.1239276
144. Yatsenko D, Nguyen T, Shen S, Gunalan K, Turner CA, Guzman R. DataJoint
elements: data work ﬂows for neurophysiology. bioRxiv (2021) 11. doi: 10.1101/
2021.03.30.437358
145. Carlson D, Carin L. Continuing progress of spike sorting in the era of big data.
Curr Opin NeuroBiol (2019) 55:90 –6. doi: 10.1016/j.conb.2019.02.007
146. Unakafova VA, Gail A. Comparing open-source toolboxes for processing and
analysis of spike and local ﬁeld potentials data. Front Neuroinform (2019) 13:57. doi:
10.3389/fninf.2019.00057
147. Markiewicz CJ, Gorgolewski KJ, Feingold F, Blair R, Halchenko YO, Miller E,
et al. The OpenNeuro resource for sharing of neuroscience data. Elife (2021) 10:e71774.
doi:10.7554/eLife.71774
148. DANDI. The DANDI archive . Available at: https://dandiarchive.org/ .
149. Gorgolewski KJ, Auer T, Calhoun VD, Craddock RC, Das S, Duff EP, et al. The
brain imaging data structure, a format for organizing and describing outputs ofneuroimaging experiments. Sci Data (2016) 3(160044):1 –9. doi: 10.1038/sdata.2016.44
150. Rübel O, Tritt A, Ly R, Dichter BK, Ghosh S, Niu L, et al. The neurodata
without borders ecosystem for neurophysiological data science. Elife (2022) 11:e78362.
doi:10.7554/eLife.78362
151. Giovannucci A, Friedrich J, Gunn P, Kalfon J, Brown BL, Koay SA, et al.
CaImAn an open source tool for scalable calcium imaging data analysis. Elife (2019) 8.
doi:10.7554/eLife.38173
152. Brain Image Library. About the brain image library . Available at: https://www.
brainimagelibrary.org/about.html .
153. Lichtman JW, P ﬁster H, Shavit N. The big data challenges of connectomics.
Nat Neurosci (2014) 17(11):1448 –54. doi: 10.1038/nn.3837
154. Wang Z, Sair HI, Crainiceanu C, Lindquist M, Bennett A, Landman BA, et al.
On statistical tests of functional connectome ﬁngerprinting. Can J Stat (2021) 49:63 –88.
doi:10.1002/cjs.11591
155. Johnson EC, Wilt M, Rodriguez LM, Norman-Tenazas R, Rivera C, Drenkow
N, et al. Toward a scalable framework for reproducible processing of volumetric,
nanoscale neuroimaging datasets. Gigascience (2020) 9:12. doi: 10.1093/gigascience/
giaa147
156. Hider RJr, Kleissas D, Gion T, Xenes D, Matelsky J, Pryor D, et al. The brain
observatory storage service and database (BossDB): a cloud-native approach forpetascale neuroscience discovery. Front Neuroinform (2022) 16(828787). doi:
10.3389/fninf.2022.828787
157. Chung J, Bridgeford E, Arroyo J, Pedigo BD, Saad-Eldin A, Gopalakrishnan V,
et al. Statistical connectomics. Annu Rev Stat Appl (2021) 8:463 –92. doi: 10.1146/
annurev-statistics-042720-023234
158. Matelsky JK, Reilly EP, Johnson EC, Stiso J, Bassett DS, Wester BA, et al.
DotMotif: an open-source tool for connectome subgraph isomorphism search andgraph queries. Sci Rep (2021) 11(1):1 –14. doi: 10.1038/s41598-021-91025-5
159. van Essen DC, Smith SM, Barch DM, Behrens TEJ, Yacoub E, Ugurbil K. The
WU-minn human connectome project: an overview. NeuroImage (2013) 80:62 –79. doi:
10.1016/j.neuroimage.2013.05.041
160. George JB, Abraham GM, Rashid Z, Amrutur B, Sikdar SK. Random neuronal
ensembles can inherently do context dependent coarse conjunctive encoding of inputstimulus without any speci ﬁc training. Sci Rep (2018) 8:1403. doi: 10.1038/s41598-018-
19462-3
161. Clawson WP, Wright NC, Wessel R, Shew WL. Adaptation towards scale-free
dynamics improves cortical stimulus discrimination at the cost of reduced detection.PloS Comput Biol (2017) 13(5):e1005574. doi: 10.1371/journal.pcbi.1005574
162. Beggs JM, Plenz D. Neuronal avalanches in neocortical circuits. J Neurosci
(2003) 3(23):11167 –77. doi: 10.1523/JNEUROSCI23-35-11167.2003
163. Edelman GM. Neural Darwinism: selection and reentrant signaling in higher
brain function. Neuron (1993) 10:115 –25. doi: 10.1016/0896-6273(93)90304-A
164. Hebb DO. Organization of behavior . New York: Wiley online library (1949).
165. Kangassalo L, Spape ́M, Ravaja N, Ruotsalo T. Information gain modulates
brain activity evoked by reading. Sci Rep (2020) 10:7671. doi: 10.1038/s41598-020-
63828-5
166. Ebitz RB, Hayden BY. The population doctrine in cognitive neuroscience.
Neuron (2021) 109:3055 –68. doi: 10.1016/j.neuron.2021.07.011
167. Ebitz RB, Tu JC, Hayden BY. Rules warp feature encoding in decision-making
circuits. PloS Biol (2020) 18:11. doi: 10.1371/journal.pbio.3000951
168. Schwartenbeck P, FitzGerald T, Mathys C, Dolan R, Kronbichler M, Friston K.
Evidence for surprise minimization over value maximization in choice behavior. Sci
Rep(2015) 5:16575. doi: 10.1038/srep16575
169. Knill DC, Pouget A. The Bayesian brain: the role of uncertainty in neural
coding and computation. Trends Neurosci (2004) 27(12):712 –9. doi: 10.1016/
j.tins.2004.10.007
170. Barlow HB. Possible principles underlying the transformations of sensory messages.
Sensory Communication (1961) 1. doi: 10.7551/mitpress/9780262518420.003.0013171. Friston KJ, Tononi G, Reeke GN, Sporns O, Edelman GM. Value-dependent
selection in the brain: simulation in a synthetic neural model. Neuroscience (1994)
59:229 –43. doi: 10.1016/0306-4522(94)90592-4
172. Madhav MS, Cowan NJ. The synergy between neuroscience and control
theory: the nervous system as inspira tion for hard control challenges. Annu Rev
Control Robot Auton Syst (2020) 3(1):243
–67. doi: 10.1146/annurev-control-060117-
104856
173. Friston KT. The free-energy principle: a uni ﬁed brain theory? Nat Rev Neurosci
(2010) 11:127 –38. doi: 10.1038/nrn2787
174. Kagan BJ, Duc D, Stevens I, Gilbert F. Neurons embodied in a virtual world:
evidence for organoid ethics? AJOB Neurosci (2022) 13(2):114 –7. doi: 10.1080/
21507740.2022.2048731
175. Khetarpal K, Riemer M, Rish I, Precup D. Towards continual reinforcement
learning: a review and perspectives. arXiv:2012.1349)v2 (2020) 78. doi: 10.48550/
arXiv.2012.13490
176. Parisi GI, Kemker R, Part JL, Kanan C, Wermter S. Continual lifelong learning
with neural networks: a review. Neural Netw (2019) 13:54 –71. doi: 10.1016/
j.neunet.2019.01.012
177. Lebedev MA, Nicolelis MAL. Brain machine interface: from basic science to
neuroprotheses and neurorehabilitation. Physiol Rev (2017) 97:767 –837. doi: 10.1152/
physrev.00027.2016
178. Bozhkov L. Deep learning models for brain machine interfaces. Ann Math Artif
Intell (2019) 90:1175 –90. doi: 10.1007/s10472-019-09668-0
179. Wahlin KJ, Maruotti JA, Sripathi SR, Ball J, Angueyra JM, Kim C, et al.
Photoreceptor outer segment-like structures in long-term 3D retinas from human
pluripotent stem cells. Sci Rep (2017) 7:1. doi: 10.1038/s41598-017-00774-9
180. Kallman A, Capowski EE, Wang J, Kaushik AM, Jansen AD, Edwards KL, et al.
Investigating cone photoreceptor development using patient-derived NRL null retinalorganoids. Commun Biol (2020) 3(1):82. doi: 10.1038/s42003-020-0808-5
181. Rost BR, Schneider-Warme F, Schmitz D, Hegemann P. Optogenetic tools for
subcellular applications in neuroscience. Neuron (2017) 96(3):572 –603. doi: 10.1016/
j.neuron.2017.09.047
182. Garita-Hernandez M, Guibbal L, Toualbi L, Routet F, Chaf ﬁol A, Winckler C,
et al. Optogenetic light sensors in human retinal organoids. Front Neurosci (2018)
12:789. doi: 10.3389/fnins.2018.00789
183. McGregor JE, Godat T, Dhakal KR, Parkins K, Strazzeri JM, Bateman BA, et al.
Optogenetic restoration of retinal ganglion cell activity in the living primate. Nat
Commun (2020) 11(1):1703. doi: 10.1038/s41467-020-15317-6
184. Cowan CS, Renner M, de Gennaro M, Gross-Scherf B, Goldblum D, Hou Y,
et al. Cell types of the human retina and its organoids at single-cell resolution. Cell
(2020) 182(6):1623 –40. doi: 10.1016/j.cell.2020.08.013
185. Gordon A, Yoon SJ, Tran SS, Makinson CD, Park JY, Andersen J, et al. Long-
term maturation of human cortical organoids matches key early postnatal transitions.Nat Neurosci (2021) 24(3):331 –42. doi: 10.1038/s41593-021-00802-y
186. Guzowski JF, McNaughton BL, Barnes CA, Worley PF. Environment-speci ﬁc
expression of the immediate-early gene arc in hippocampal neuronal ensembles. Nat
Neurosci
(1999) 2:1120 –4. doi: 10.1038/16046
187. Xiao MF, Roh SE, Zhou J, Chien CC, Lucey BP, Craig MT, et al. A biomarker-
authenticated model of schizophrenia implicating NPTX2 loss of function. Sci Adv
(2021) 7(48). doi: 10.1126/sciadv.abf6935
188. World Health Organization. Dementia (2021). Available at: https://www.who.
int/news-room/fact-sheets/detail/dementia .
189. Nichols E, Steinmetz JD, Vollset SE, Fukutaki K, Chalek J, Abd-Allah F, et al.
Estimation of the global prevalence of dementia in 2019 and forecasted prevalence in2050: an analysis for the global burden of disease study 2019. Lancet Public Health
(2022) 7(2):e105 –25. doi: 10.1016/S2468-2667(21)00249-8
190. Centers for Disease Control and Prevention. Leading causes of death (2021).
Available at: https://www.cdc.gov/nchs/fastats/leadingcauses-of-death.htm .
191. Xu J, Zhang Y, Qiu C, Cheng F. Global and regional economic costs of
dementia: a systematic review. Lancet (2017) 390. doi: 10.1016/S0140-6736(17)33185-9
192. World Health Organization. Global status report on the public health response
to dementia . WHO Geneva: World Health Organization (2021).
193. Pistollato F, Ohayon EL, Lam A, Langley GR, Novak TJ, Pamies D, et al.
Alzheimer Disease research in the 21st century: past and current failures, newperspectives and funding priorities. Oncotarget (2016) 7(26):38999 –9016. doi:
10.18632/oncotarget.9175
194. Drummond E, Wisniewski T. Alzheimer ’s disease: experimental models and
reality. Acta Neuropathol (2017) 133(2):155 –75. doi: 10.1007/s00401-016-1662-x
195. Maenner MJ, Shaw KA, Bakian AV, Bilder DA, Durkin MS, Esler A, et al.
Prevalence and characteristics of autism spectrum disorder among children aged 8years –autism and developmental disabilities monitoring network, 11 sites,
united states, 2018. MMWR Surveill Summ (2021) 70(11):1 –16. doi: 10.15585/
mmwr.ss7011a1
196. Zhong X, Harris G, Smirnova L, Zufferey V, de Cassia da Silveira E Sa ́R,
Baldino Russo F, et al. Antidepressant paroxetine exerts developmental neurotoxicity inan iPSC-derived 3D human brain model. Front Cell Neurosci (2020) 14(25). doi:
10.3389/fncel.2020.00025Smirnova et al. 10.3389/fsci.2023.1017235
Frontiers in Science frontiersin.org 21